<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutWindow</class>
 <widget class="QMainWindow" name="CheckoutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 108, 78);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="25,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="OtherLabel3">
          <property name="font">
           <font>
            <family>Vineta BT</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(208, 208, 208);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Tom's Grill&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="indent">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="" native="true">
        <property name="minimumSize">
         <size>
          <width>336</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="OtherLabel1">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Your Receipt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="OtherLabel2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;This receipt becomes official once paid.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <item>
        <widget class="QGroupBox" name="">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="OtherLabel5">
            <property name="font">
             <font>
              <family>Amsterdam Traveling</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Your Orders</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 204, 158);
border-radius: 10px
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="" native="true">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="Logo">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>300</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/logo/pictures/logo 4.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PaymentMethodLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Payment Method</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="PaymentMethod">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 204, 158);

</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="DeliveryMethodLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Dine In / Take Out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="DeliveryMethod">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 204, 158);

</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PromoCodeLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Promo Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PromoCode">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 204, 158);

</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PromoCodeApplyButton">
            <property name="font">
             <font>
              <family>Georgia</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,25">
      <item>
       <widget class="QWidget" name="" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(250, 240, 202);
border-radius: 10px
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="AdditionalChargeLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Additional Charges</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AdditionalCharge">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VATLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>VAT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VAT">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="DiscountsLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Discounts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Discounts">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="GrandTotalLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Grand Total:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="GrandTotal">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlacePrderButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 204, 158);
color: rgb(89, 52, 27);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Place Order</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
